docs/docs/running-experiments/running-holdouts.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Holdouts
+description: Run holdout experiments to measure the long-term impact of features
+slug: /app/holdouts
+---
+
+Holdout experiments (or simply "holdouts") measure the long-term impact of features by maintaining a control group that doesn't receive new functionality. While most users experience your latest features and improvements, a small percentage remains on the original version, providing a baseline to measure cumulative effects over time.
+
+Learn more about holdouts in our [Knowledge Base](/kb/experiments/holdouts).
+
+## How Holdouts Work in GrowthBook
+
+In GrowthBook, users in the holdout treatment group receive the same experience as the general population: they participate in experiments, receive feature releases, and see all your latest changes. The holdout control group, however, stays on the baseline experience.
+
+This approach measures the cumulative impact of all changes over time by comparing the general population to the holdout control group.
+
+## Creating a Holdout
+
+![Add Holdout](/images/holdouts-add-modal.png)
+
+1. Navigate to **Experiments** → **Holdouts** and click **Add Holdout**
+
+2. Configure your holdout settings:
+
+- **Name**: Choose a descriptive name (e.g., `q1-2025-product-holdout`)
+- **Holdout Size**: Defaults to 5% (meaning 5% of users remain in the control group)
+- **Description**: Document the purpose and scope of your holdout
+- **Key Metric**: Select a metric to track the impact of the holdout
+
+3. Click **Start Holdout** to begin
+
+### Adding Experiments to a Holdout
+
+![Add Holdout to an experiment](/images/holdouts-experiment-with-holdout.png)
+
+Once created, you can use your holdout in new experiments:
+
+1. When creating a new experiment, you'll see a **Holdout** field
+2. Your active holdout will selected in the dropdown by default
+3. Complete the rest of your experiment setup normally
+
+The experiment will run as usual, with the holdout tracking long-term cumulative effects in the background.
+
+## Holdout Lifecycle
+
+### Active Period
+
+During the active period (typically a quarter), you'll:
+
+- Continue adding new experiments to the holdout
+- Launch features as normal
+- Let the holdout accumulate data on cumulative impact
+
+### Analysis Period
+
+After your planned duration, transition to the analysis period:
+
+1. **Stop adding new experiments**: The holdout enters analysis mode where no new experiments can be added
+2. **Extended measurement**: Continue measuring for 2-4 weeks to capture delayed effects and reduce statistical noise
+3. **Dynamic lookback window**: GrowthBook automatically applies a lookback window based on when each experiment was added, ensuring clean measurement periods
+
+During analysis, the system:
+
+Retains existing user assignments to avoid rebucketing
```

Comment:
```suggestion
Continues to split traffic into your holdout and general population groups
```

Much clearer, the other looks like it uses sticky bucketing or some kind of data store.